beta-src/src/components/ui/WDMoveControls.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+
+import { Stack } from "@mui/material";
+import WDButton from "./WDButton";
+import MoveStatus from "../../types/MoveStatus";
+import Move from "../../enums/Move";
+
+interface WDMoveControlsProps {
+  gameState: MoveStatus;
+  setState: (move: Move) => void;
+}
+const WDMoveControls: React.FC<WDMoveControlsProps> = function ({
+  gameState,
+  setState,
+}) {
+  const { ready } = gameState;
+
+  return (
+    <Stack direction="row" spacing={2} alignItems="center">
+      <WDButton
+        color="primary"
+        disabled={ready}
+        onClick={() => setState(Move.SAVE)}
+      >
+        Save
+      </WDButton>
+      <WDButton color="primary" onClick={() => setState(Move.READY)}>
```

Comment:
this function/prop should be called `toggleState` since you're not specifying what you're setting and toggling the ready/unready states.